Number_types/include/CGAL/GMP/Gmpfi_type.h

```diff
@@ -618,8 +618,13 @@ Gmpfi Gmpfi::kthroot(int k,Gmpfi::Precision_type p)const{
         // MPFI does not provide k-th root functions
         CGAL_assertion(p>=MPFR_PREC_MIN&&p<=MPFR_PREC_MAX);
         Gmpfi result (0, p);
-        mpfr_root(&(result.mpfi())->left, left_mpfr(), k,GMP_RNDD);
-        mpfr_root(&(result.mpfi())->right,right_mpfr(),k,GMP_RNDU);
+        #if(MPFR_VERSION_MAJOR < 4)
+            mpfr_root(&(result.mpfi())->left, left_mpfr(), k,GMP_RNDD)
```

Comment:
@kkatrio  Add a semicolon please.

Reply:
I just did it in https://github.com/CGAL/cgal/pull/2808/commits/47307b4bfbb4adb5ddd04c7a15155a256bae1293.